pom.xml

```diff
@@ -81,6 +81,37 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
+                <version>3.0.5</version>
```

Comment:
Is it better to create a property for this version as for the other dependencies and plugins ?

Reply:
Can do if you want, but it's not something I would expect submodules to override (unlike dependencies, which is where having a version property is super-useful)